src/walletHandlers/messageHandler.js

```
@@ -0,0 +1,23 @@
+
+eventBus.on('text', async (from_address, data) => {
+    const unlock = await mutex.lock(from_address);
+
+    try {
+        if (data.trim().startsWith("[Signed message]")) { // User send signed message
+            await verifyHandler(from_address, data);
+        } else if (Validation.isWalletAddress(String(data).trim()) && walletSessionStore.getSession(from_address)) { // User send wallet address
+            await walletAddressHandler(from_address, String(data).trim());
+        } 
+    } finally {
+        return unlock();
```

Biome (1.9.4)

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)